snippets/audit-logs-default.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies and modules:
+//
+// * logging/log_collection_forwarding/configuring-log-forwarding.adoc
+//
+// * modules/cluster-logging-collector-log-forwarding-about.adoc
```

Comment:
I can't find an include statement for this snippet in the module listed here. Should you either include in snippet in this module or remove the comment line here?